### ISSUE TITLE
Has trl impelmented ppo-ptx?

### ISSUE THREAD

**Issue by skepsun** [huggingface/trl#716]
I found DeepSpeed-chat has come with ppo-ptx, which utilizes an extra unsupervised (or pretrain) loss based on a unsupervised dataloader (pretrain data, e.g., wikitext-103). It can effectively prevent actor from outputing nothing (but its rewards still increase).

**Comment by lvwerra** [comment]
No, it's currently not implemented, but it could be a nice addition. Haven't encountered the issue before where the model doesn't output anything..

**Comment by skepsun** [comment]
Thanks. I hope this feature can be finally implemented.
